src/transaction/log_tran_table.c

```diff
@@ -5042,7 +5042,7 @@ logtb_reflect_global_unique_stats_to_btree (THREAD_ENTRY * thread_p)
        stats = (GLOBAL_UNIQUE_STATS *) lf_hash_iterate (&it))
     {
       /* reflect only if some changes were logged */
-      if (!LSA_ISNULL (&stats->last_log_lsa))
+      if (log_is_no_logging () || !LSA_ISNULL (&stats->last_log_lsa))
```

Comment:
I wonder `btree_reflect_global_unique_statistics()` works well even though `stats->last_log_lsa` is NULL. I don't have enough background of this routine. Could you please explain in detail?

Reply:
In **logtb_update_global_unique_stats_by_delta**, the **last_log_lsa** of stats is copying the lsa of the transaction by LSA_COPY (&stats->last_log_lsa, &tdes->tail_lsa).
This is to check with log's lsa that stats have changed.
However, in the case of no-logging, it is not possible to know whether changes have been made through lsa, so statistical information must always be reflected.